packages/framework-provider-aws/src/index.ts

```diff
@@ -51,12 +55,19 @@ export const Provider: ProviderLibrary = {
   // ProviderAuthLibrary
   auth: {
     rawToEnvelope: rawSignUpDataToUserEnvelope,
+    fromAuthToken: userEnvelopeFromAuthToken.bind(null, userPool),
   },
   // ProviderAPIHandling
   api: {
     requestSucceeded,
     requestFailed,
   },
+  connections: {
+    storeData: storeConnectionData.bind(null, dynamoDB),
+    fetchData: fetchConnectionData.bind(null, dynamoDB),
+    deleteData: deleteConnectionData.bind(null, dynamoDB),
+    sendMessage: sendMessageToConnection,
+  },
```

Comment:
This is the new namespace `connections` with functions to deal with... wair for it... connections!